src/main/java/com/mercadolivre/bootcamp/projeto_integrador/controller/RepresentativeController.java

```diff
@@ -0,0 +1,47 @@
+package com.mercadolivre.bootcamp.projeto_integrador.controller;
+
+import com.mercadolivre.bootcamp.projeto_integrador.entity.Representative;
+import com.mercadolivre.bootcamp.projeto_integrador.service.RepresentativeServiceImpl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+public class RepresentativeController {
+    RepresentativeServiceImpl representativeServiceImpl;
+
```

Comment:
Alterar a linha 13 para 'RepresentativeService  representativeService' para consumir os métodos da interface do serviço e não da implementação dele diretamente. Acrescentar a anotação @Autowired antes da linha 13 para autorizar a injeção de dependências pelo Spring.

Reply:
Alterações realizadas. Obs.: Ao acrescentar @Autowired, a IDE retorna a seguinte notificação de warning: "Field injection is not recommended".